h/static/scripts/group-forms/hooks/use-group-annotations.tsx

```diff
@@ -32,6 +32,16 @@ export type GroupAnnotationsResult = {
    * first chunk when invoked.
    */
   loadNextPage: () => void;
+
+  /**
+   * A callback to update the moderation status of one annotation from the list.
+   * If the new status does not match current filter status, the annotation will
```

Comment:
There should be a blank line after the summary line. Is this method going to make a call to the server in future or is it always going to only change the status of the local copy? The description should make that clear.